index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -158,6 +163,8 @@ void shouldReturnAggregationsWithExpectedCount(Entry<String, Integer> entry)
                                     DEFAULT_OFFSET_SIZE, DEFAULT_QUERY_SIZE);
         var docCount = getDocCount(searchResponse, entry.getKey());
 
+        assertThat(getDocCount(searchResponse, PUBLICATION_DATE_AGG), is(not(nullValue())));
```

Comment:
Assert that `PUBLICATION_DATE_AGG` is equal to some expected value instead? To insure that the aggregation works as expected?